sidebarsDocs.js

```diff
@@ -72,1187 +71,1284 @@ module.exports = {
       ],
       collapsed: true,
     },
     {
-      type: 'category',
-      label: 'Open Source',
-      items: ['open-source/open-source'],
+      type: "category",
+      label: "Open Source",
+      items: ["open-source/open-source"],
       collapsed: true,
     },
     {
-      type: 'category',
-      label: 'FAQ',
-      items: ['faq/helium-network', 'faq/security', 'faq/hotspot-manufacturers', 'faq/build-on-network', 'faq/data-credits'],
+      type: "category",
+      label: "FAQ",
+      items: [
+        "faq/helium-network",
+        "faq/security",
+        "faq/hotspot-manufacturers",
+        "faq/build-on-network",
+        "faq/data-credits",
+      ],
       collapsed: true,
     },
     {
-      type: 'category',
-      label: 'Troubleshooting',
-      items: ['troubleshooting/hotspot-setup-troubleshooting', 'troubleshooting/replace-sd-card', 'troubleshooting/improving-poc', 'troubleshooting/network-troubleshooting', 'troubleshooting/hotspot-led','troubleshooting/understanding-witnesses'],
+      type: "category",
+      label: "Troubleshooting",
+      items: [
+        "troubleshooting/hotspot-setup-troubleshooting",
+        "troubleshooting/replace-sd-card",
+        "troubleshooting/improving-poc",
+        "troubleshooting/network-troubleshooting",
+        "troubleshooting/hotspot-led",
+        "troubleshooting/understanding-witnesses",
+      ],
       collapsed: true,
     },
   ],
   api: [
-    'api/home',
-   {
-      type: 'category',
-      label: 'Blockchain',
-      items: ['api/blockchain/introduction', 'api/blockchain/stats', 'api/blockchain/blocks', 'api/blockchain/accounts', 'api/blockchain/validators', 'api/blockchain/hotspots', 'api/blockchain/cities', 'api/blockchain/locations', 'api/blockchain/transactions', 'api/blockchain/pending-transactions', 'api/blockchain/oracle-prices', 'api/blockchain/chain-variables', 'api/blockchain/ouis', 'api/blockchain/rewards', 'api/blockchain/dc-burns', 'api/blockchain/challenges', 'api/blockchain/elections', 'api/blockchain/state-channels', 'api/blockchain/assert-locations'],
+    "api/home",
+    {
+      type: "category",
+      label: "Blockchain",
+      items: [
+        "api/blockchain/introduction",
+        "api/blockchain/stats",
+        "api/blockchain/blocks",
+        "api/blockchain/accounts",
+        "api/blockchain/validators",
+        "api/blockchain/hotspots",
+        "api/blockchain/cities",
+        "api/blockchain/locations",
+        "api/blockchain/transactions",
+        "api/blockchain/pending-transactions",
+        "api/blockchain/oracle-prices",
+        "api/blockchain/chain-variables",
+        "api/blockchain/ouis",
+        "api/blockchain/rewards",
+        "api/blockchain/dc-burns",
+        "api/blockchain/challenges",
+        "api/blockchain/elections",
+        "api/blockchain/state-channels",
+        "api/blockchain/assert-locations",
+      ],
       collapsed: true,
     },
-    'api/console',
+    "api/console",
```

Comment:
I've seen in a couple of our repos we prefer `single-quote`. Should we stick to that convention?

Reply:
I'm happy to do so! default prettier settings use double, but with the inclusion of a `.prettierrc` that's an easy override to conform to other repos